src/main/controllers/SubscriptionManagementController.ts

```diff
@@ -1,17 +1,19 @@
-import {PipRequest} from '../models/request/PipRequest';
+import { PipRequest } from '../models/request/PipRequest';
 import { Response } from 'express';
 import { SubscriptionService } from '../service/subscriptionService';
-import {cloneDeep} from 'lodash';
+import { cloneDeep } from 'lodash';
 
 const subscriptionService = new SubscriptionService();
 
 export default class SubscriptionManagementController {
-  public get(req: PipRequest, res: Response): void {
+  public async get(req: PipRequest, res: Response): Promise<void> {
     if (req.user) {
-      // currently only 2 users are mocked, userId: 1 has subscriptions, userId: 2 doesnt
-      const userId = req.user['id'] === '1' ? 1 : 2;
-      const caseTableData = subscriptionService.generateCaseTableRows(userId);
-      const courtTableData = subscriptionService.generateCourtTableRows(userId);
+      const tableData = {
+        cases: await subscriptionService.generateCaseTableRows(req.user['id']),
+        courts: await subscriptionService.generateCourtTableRows(req.user['id']),
+      };
+      const caseTableData = tableData.cases;
+      const courtTableData = tableData.courts;
```

Comment:
could this not just be const caseTableData = await subscriptionService.generateCaseTableRows(req.user['id']); and same with courts rather than a variable assign that adds no value?

Reply:
done